app/api/resources/users.py

```diff
@@ -52,8 +59,66 @@ def post(cls):
         result = post_request(f"{BASE_MS_API_URL}/register", data)
 
         if result[1] == HTTPStatus.OK:
-            return f"{messages.USER_WAS_CREATED_SUCCESSFULLY}", HTTPStatus.CREATED
+            return messages.USER_WAS_CREATED_SUCCESSFULLY, HTTPStatus.CREATED
         elif result[1] == HTTPStatus.INTERNAL_SERVER_ERROR:
-            return f"{messages.INTERNAL_SERVER_ERROR}", HTTPStatus.INTERNAL_SERVER_ERROR
+            return messages.INTERNAL_SERVER_ERROR, HTTPStatus.INTERNAL_SERVER_ERROR
         else:
             return result
+
+
+@users_ns.route("login")
+class LoginUser(Resource):
+    @classmethod
+    @users_ns.doc("login")
+    @users_ns.response(HTTPStatus.OK, "Successful login", login_response_body_model)
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST,
+        f"{messages.USERNAME_FIELD_IS_MISSING}\n" 
+        f"{messages.PASSWORD_FIELD_IS_MISSING}",
+    )
+    @users_ns.response(HTTPStatus.UNAUTHORIZED, f"{messages.WRONG_USERNAME_OR_PASSWORD}")
+    @users_ns.response(HTTPStatus.FORBIDDEN, f"{messages.USER_HAS_NOT_VERIFIED_EMAIL_BEFORE_LOGIN}")
+    @users_ns.response(
+        HTTPStatus.INTERNAL_SERVER_ERROR, f"{messages.INTERNAL_SERVER_ERROR}"
+    )
+    @users_ns.expect(login_request_body_model)
+    def post(cls):
```

Comment:
This method is too big and does too many things. Break this down into multiple smaller functions 
https://martinfowler.com/bliki/FunctionLength.html

Reply:
I'm not sure I'm following. Isn't this just to cover POST /login and its responses? If you want to make it into smaller functions, can you please give me example of endpoints that will come as the result of slicing them?

Reply:
Done. Thanks for giving this idea, @ramitsawhney27 . I've refactored HTTP response checking for both Register and Login. Hopefully this will help tidying up responses for any future API endpoints too.

Reply:
Note that for `all other errors` comment under http_response_checker function, that is only for errors that are unplanned. As we move through other functionalities, I'll keep adding smaller functions to check HTTP response type/message.